tests: added teamcity runner

i582 opened the pull request:
## Why

To display tests in a convenient way inside the JetBrains IDE. See next video for example:

https://user-images.githubusercontent.com/51853996/207796346-c4fb7acc-8168-4fed-9ba3-070161f69fb4.mov


References:

- https://www.jetbrains.com/help/teamcity/service-messages.html

Fixes https://github.com/i582/vlang-idea/issues/29

spytheman commented:
Looks amazing!

spytheman commented:
Please also add an entry in `cmd/v/help/test.txt` . That file is shown by `v help test`.

i582 commented:
> Please also add an entry in `cmd/v/help/test.txt` . That file is shown by `v help test`.

Added.

> Please also add an entry in `cmd/v/help/test.txt` . That file is shown by `v help test`.

Added
